lib/std/c/openbsd.zig

```diff
@@ -1615,39 +1593,3 @@ pub const KERN = struct {
     pub const PROC_ENV = 3;
     pub const PROC_NENV = 4;
 };
-
-pub const HW = struct {
-    pub const MACHINE = 1;
-    pub const MODEL = 2;
-    pub const NCPU = 3;
-    pub const BYTEORDER = 4;
-    pub const PHYSMEM = 5;
-    pub const USERMEM = 6;
-    pub const PAGESIZE = 7;
-    pub const DISKNAMES = 8;
-    pub const DISKSTATS = 9;
-    pub const DISKCOUNT = 10;
-    pub const SENSORS = 11;
-    pub const CPUSPEED = 12;
-    pub const SETPERF = 13;
-    pub const VENDOR = 14;
-    pub const PRODUCT = 15;
-    pub const VERSION = 16;
-    pub const SERIALNO = 17;
-    pub const UUID = 18;
-    pub const PHYSMEM64 = 19;
```

Comment:
`PHYSMEM64` is used by `totalSystemMemory()` (in lib/std/process.zig)

Reply:
Incorrectly reverted by @devnexen in this PR; not reverted in #16636.